<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1588</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxLoadImage">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>281</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>Load Image</string>
    </property>
    <widget class="QPushButton" name="pushButtonLoadFolder">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadImageR">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_R</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadImageL">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_L</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxStereoDisparityMap">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>140</y>
      <width>281</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Stereo Disparity Map</string>
    </property>
    <widget class="QPushButton" name="pushButtonStereoDisparityMap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 Stereo Dispparity Map</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxCalibration">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>281</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Calibration</string>
    </property>
    <widget class="QPushButton" name="pushButtonFindCorners">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 Find Corners</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFindIntrinsic">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2 Find Intrinsic</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxFindExtrinsic">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>1.3 Find Extrinsic</string>
     </property>
     <widget class="QPushButton" name="pushButtonFindExtrinsic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>1.3 Find Extrinsic</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxFindExtrinsic">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>110</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonShowResult">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>540</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1.5 Show Result</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFindDistortion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1.4 Find Distortion</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAugmentedReality">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>281</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Augmented Reality</string>
    </property>
    <widget class="QPushButton" name="pushButtonShowWordsOnBoard">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Show Words on Board</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditAugmentedReality">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowWordsVertically">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Show Words Vertically</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
